apps/modernization-ui/src/apps/page-builder/generated/models/Page_PageSummary_.ts

```diff
@@ -7,16 +7,15 @@ import type { PageSummary } from './PageSummary';
 import type { Sort } from './Sort';
 
 export type Page_PageSummary_ = {
-    content?: Array<PageSummary>;
-    empty?: boolean;
-    first?: boolean;
-    last?: boolean;
-    number?: number;
-    numberOfElements?: number;
-    pageable?: Pageable;
-    size?: number;
-    sort?: Sort;
-    totalElements?: number;
-    totalPages?: number;
+    content: Array<PageSummary>;
+    empty: boolean;
+    first: boolean;
+    last: boolean;
+    number: number;
+    numberOfElements: number;
+    pageable: Pageable;
+    size: number;
+    sort: Sort;
+    totalElements: number;
+    totalPages: number;
```

Comment:
Info: Was this a manual change? If so this file is an auto-generated file. We would need to add `@ApiModelProperty(required = true)` to the corresponding java object and trigger the generation.

Reply:
Yeah this was a manual change, thanks for the info. I just kinda figured these values would always be returned.  Should I revert or is it ok to change the @apiModelProperty to true?